docs/ota-client-guide/modules/ROOT/pages/evaluation-to-prod.adoc

```diff
@@ -30,6 +37,8 @@ include::partial$recommended-steps.adoc[tags=evaluate-steps]
 
 == Build your own OTA-enabled solution
```

Comment:
This need to match the phases listed above

Reply:
Good call, thanks. Fixed.